tests/cli/utils.py

```diff
@@ -4,6 +4,10 @@
 from click.testing import Result
 
 VALIDATION_OPERATORS_DEPRECATION_MESSAGE: str = "Your data context with this configuration version uses validation_operators, which are being deprecated."
+LEGACY_CONFIG_DEFAULT_CHECKPOINT_STORE_MESSAGE: str = (
```

Comment:
Curious: why is this message being changed?  More importantly, some tests relied on it; do they still pass?